engine-plugins/frame-plugins/src/main/scala/org/trustedanalytics/atk/engine/frame/plugins/load/JdbcPlugin/LoadFromJdbcPlugin.scala

```diff
@@ -0,0 +1,75 @@
+/*
+// Copyright (c) 2015 Intel Corporation 
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+package org.trustedanalytics.atk.engine.frame.plugins.load.JdbcPlugin
+
+import org.apache.spark.frame.FrameRdd
+import org.trustedanalytics.atk.domain.frame.load.{ JdbcArgs }
+import org.trustedanalytics.atk.domain.frame.{ FrameEntity }
+import org.trustedanalytics.atk.domain.schema.{ Column, FrameSchema }
+import org.trustedanalytics.atk.engine.frame.SparkFrame
+import org.trustedanalytics.atk.engine.frame.plugins.load.LoadRddFunctions
+import org.trustedanalytics.atk.engine.plugin.{ Invocation, PluginDoc, SparkCommandPlugin }
+
+import spray.json._
+import org.trustedanalytics.atk.domain.DomainJsonProtocol._
+
+/**
+ * Parsing data to load and append to data frames
+ */
+@PluginDoc(oneLine = "Append data from a Jdbc table into an existing (possibly empty) FrameRDD",
+  extended = "Append data from a Jdbc table into an existing (possibly empty) FrameRDD",
+  returns = "the initial FrameRDD with the Jdbc data appended")
```

Comment:
These comments should be directed towards an end-user.  Mentioning FrameRDD isn't meaningful --just say "Frame"